### ISSUE TITLE
Feature: Merge LAIT into DfE shiny-template

### ISSUE THREAD

**Issue by JT-39** [dfe-analytical-services/local-authority-interactive-tool#6]
**Feature Overview**

The work on converting the Local Authority Interactive Tool (LAIT) to a shiny app began in Azure DevOps. This decision was taken to (a) see how feasible the project would be and (b - mainly) a lack of knowledge regarding what data/ functionality of the app that can be shared publicly.

Since then, these data sharing issues have been clarified and solved. Hence, the work done on LAIT in DevOps is ready to be migrated to GitHub (this was always going to happen at some point as LAIT is a public facing tool).

The DfE `shiny-template` was not used initially to reduce the complexity of the project so that other analysts with less coding experience stood a better chance at understanding the work. However, now this phase is over and the code is ready to go onto GitHub the template and LAIT work need to be merged.

TLDR; The merger of the DfE `shiny-template` and prior LAIT development done in DevOps.

**Problem**

The LAIT development so far has been done in DevOps and has not used the DfE `shiny-template`. This template is required for the app to go onto the public facing Shiny server.

**Proposed Solution**

- I am going to branch off the `main` (which contains the DfE `shiny-template` + a few edits from me to suit my styling/ best practice) into a branch named `shiny-template-lait-merge`.
- Then using the branch `prepare-la-level-azure`, where I have prepped the prior LAIT code for a merger (styling, linting, ...), merge this branch into `shiny-template-lait-merge`.
- Address all merge conflicts in R. Keeping a blend of the DfE `shiny-template` and the prior LAIT development.
- Merge this back into `main`, with hopefully not too many conflicts (but if there are I should be able to take all new changes OR I simply replace `main` with `shiny-template-lait-merge`).

**Impact**

This will remove all analytical functionality from the DfE `shiny-template`, but keep all generic features (cookies, accessibility statement) and styling (using `{shinyGovstyle}`). For the LAIT work, all functionality should be replicated as a minimum. Some custom styling will be integrated such as using `{bslib}` for the app infrastructure.

**Additional Information**

Possibility that the full conversion to `{shinyGovstyle}` may happen in an issue, branch and pull request after this issue. Mainly due to uncertainty about how (if) `{bslib}` is integrated into the package. This package and its shiny infrastructure is what will be behind the core of this app due to it being advocated for by Posit, the community and achieving high levels of accessibility, in other words it is best practice,
